tests/ConfigTest.php

```diff
@@ -133,7 +133,7 @@ public function testThatCustomSymfonyFinderWorks()
         $config = Config::create()->setFinder($finder);
 
         $iterator = $config->getFinder()->getIterator();
-        $this->assertSame(1, count($iterator));
+        $this->assertCount(1, $iterator);
```

Comment:
yeah :(
Wonder if HHVM is not hit by same bug as PHP was;
https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Finder/Iterator/FilterIterator.php#L32